spec/requests/api/v3/work_packages/available_relation_candidates_resource_spec.rb

```diff
@@ -197,10 +197,17 @@ def work_packages
       end
 
       it 'does not return work packages from that project' do
-        # TODO: Remove and fix before merge
-        pending "Temporary disable"
```

Comment:
This test was broken in 13.1 and we disabled it for now, it sort of was testing exactly for this behavior. But because `subjects` was just filled with `[nil, nil]` entries we have temporarily disabled the tests.

I have changed the entire test file to work with the IDs instead of the names, this would have given us a hint what the `nil` values were and we might have figured this out sooner

Reply:
Nice.